rct229/ruletest_engine/ruletest_jsons/section23/rule_23_8.json

```diff
@@ -0,0 +1,392 @@
+                                                                {
+                                                                    "airflow": 28.31684659199999,
+                                                                    "result": 41
+                                                                },
+                                                                {
+                                                                    "airflow": 33.03632102399999,
+                                                                    "result": 54
+                                                                },
+                                                                {
+                                                                    "airflow": 37.75579545599999,
+                                                                    "result": 68
+                                                                },
+                                                                {
+                                                                    "airflow": 42.475269887999985,
+                                                                    "result": 83
+                                                                },
+                                                                {
+                                                                    "airflow": 47.194744319999984,
+                                                                    "result": 100
+                                                                }
+                                                            ]
+                                                        }
+                                                    }
+                                                ]
+                                            },
+                                            {
+                                                "id": "Thermal Zone 2",
+                                                "thermostat_cooling_setpoint_schedule": "Required Cooling Schedule 1",
+                                                "thermostat_heating_setpoint_schedule": "Required Heating Schedule 1",
+                                                "terminals": [
+                                                    {
+                                                        "id": "VAV Air Terminal 2",
+                                                        "is_supply_ducted": true,
+                                                        "type": "VARIABLE_AIR_VOLUME",
+                                                        "served_by_heating_ventilating_air_conditioning_system": "System 7",
+                                                        "heating_source": "HOT_WATER",
+                                                        "heating_from_loop": "Boiler Loop 1"
+                                                    }
```

Comment:
I believe we also need ```output_validation_points``` key here as well. Could you double-check this?

Reply:
Juan, I just realized this.

I believe the ```output_validation_points``` should be located under the ```fan_system``` not under the ```fan``` in the ```terminals```. Could you verify and address this?

Reply:
Hi Yun Joon,

Per the schema, the output validation points are at the Fan and not FanSystem level:

https://github.com/pnnl/ruleset-checking-tool/blob/develop/rct229/schema/ASHRAE229.schema.json#L2716-L2721

Reply:
@jugonzal07 

There can be the ```supply_fans``` key under the ```fan_system``` key and ```supply_fans``` key can also have the ```output_validation_points``` key, which is from the [23-8 RDS](https://github.com/pnnl/ruleset-checking-tool/blob/develop/docs/section23/Rule23-8.md#:~:text=fan_b.output_validation_points) - fan_b is from the ```supply_fans```.

Please let me know if this doesn't make sense!

Reply:
My comment from yesterday was wrong. I should've said that the ```output_validation_points``` should be located under the ```supply_fans``` under the ```fan_system```. My apologies!

Reply:
Understood! Yes I see what you mean now.

I have pushed an update. You should see the fan_system supply_fans utilizing these output validation points and not the terminal fan.